### PR TITLE
Extract and use real-world data in tests

### PR DESCRIPTION
**Pull request by adam-cattermole**
* Also update the cel-interpreter explicitly to grab this change (https://github.com/clarkmcc/cel-rust/pull/106)
* The data we expect when sending GRPC requests needed updating as there were changes to the `request.scheme`, the `request.time` as well as the ordering of some headers

### CODE REVIEW

File: src/configuration.rs [hunk 1]
```diff
@@ -10,7 +10,7 @@ use crate::data;
 use crate::data::PropertyPath;
 use crate::data::{AttributeValue, Predicate};
 use crate::service::GrpcService;
-use cel_interpreter::functions::duration;
+use cel_interpreter::functions::time::duration;
```
Comment (alexsnaps) [inline src/configuration.rs:13]: Ugh... yeah! Blame me for that one 😢

File: src/data/cel.rs [hunk 2]
```diff
@@ -441,6 +442,22 @@ fn properties<'e>(exp: &'e CelExpression, all: &mut Vec<Vec<&'e str>>, path: &mu
     }
 }
 
+#[allow(dead_code)]
```
Comment (alexsnaps) [inline src/data/cel.rs:445]: I'd put this behing a feature flag or something... if only as `cfg[test]`... wdyt?

File: tests/util/data.rs [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+}
+
+pub mod connection {
+    pub const ID: &[u8] = &[0, 0, 0, 0, 0, 0, 0, 0];
+    pub const MTLS: &[u8] = &[0];
+}
+
+// Example CheckRequest
+#[allow(dead_code)]
+const CHECK_REQUEST: &str = r#"
```
Comment (alexsnaps) [inline tests/util/data.rs:64]: Is there any use for these two to actually be `const`? How would I use `CHECK_REQUEST`?

Comment (adam-cattermole) [reply tests/util/data.rs:64]: At the moment they're not used and I guess could just be comments

File: src/data/cel.rs (hunk 2 above)
Comment (adam-cattermole) [reply src/data/cel.rs]: Done

File: tests/util/data.rs (hunk 3 above)
Comment (adam-cattermole) [reply tests/util/data.rs:64]: Do you think these are better commented out? or perhaps in a non-source file

Comment (alexsnaps) [reply tests/util/data.rs:64]: I don't care really... mostly wondered tbh.

**Review by alexsnaps — APPROVED** [review]

**Review by didierofrivia — APPROVED** [review]
🥇

**Pull request merged**